src/test/java/teammates/storage/api/FeedbackResponseMonitorDbTest.java

```diff
@@ -0,0 +1,58 @@
+package teammates.storage.api;
+
+import java.util.List;
+
+import org.testng.annotations.Test;
+
+import teammates.common.datatransfer.attributes.FeedbackResponseRecordAttributes;
+import teammates.common.exception.EntityAlreadyExistsException;
+import teammates.common.exception.InvalidParametersException;
+import teammates.test.BaseComponentTestCase;
+
+/**
+ * SUT: {@link FeedbackResponseMonitorDb}.
+ */
+public class FeedbackResponseMonitorDbTest extends BaseComponentTestCase {
+    FeedbackResponseMonitorDb db = new FeedbackResponseMonitorDb();
+    long[] countTestData = new long[] { 105, 105, 105, 105, 105 };
+
+    private void populateTestData() throws EntityAlreadyExistsException, InvalidParametersException {
+        long currentTime = System.currentTimeMillis();
+        for (int i = 0; i < countTestData.length; i++) {
+            long timestamp = currentTime - 120 * i;
+            FeedbackResponseRecordAttributes attributes = new FeedbackResponseRecordAttributes(countTestData[i], timestamp);
+            db.createEntity(attributes);
+        }
+    }
+
+    @Test
+    public void testResponseRecordsInterval() throws EntityAlreadyExistsException, InvalidParametersException {
+        long interval = 120;
+        db.purgeResponseRecords();
+        populateTestData();
+        //get all records with interval 120 milliseconds
```

Comment:
Add some empty lines to separate logical blocks.
E.g. line 31-32 clearly are just init logic, why not separate out as 2 lines block?